lib/embedded_ansible/appliance_embedded_ansible.rb

```diff
@@ -70,6 +71,32 @@ def api_connection
     api_connection_raw("localhost", HTTP_PORT)
   end
 
+  def create_local_playbook_repo
+    self.class.consolidate_plugin_playbooks(playbook_repo_path)
+
+    Dir.chdir(playbook_repo_path) do
+      require 'rugged'
+      repo = Rugged::Repository.init_at(".")
+      index = repo.index
+      index.add_all("*")
+      index.write
+
+      options              = {}
+      options[:tree]       = index.write_tree(repo)
+      options[:author]     = options[:committer] = { :email => "system@localhost", :name => "System", :time => Time.now.utc }
+      options[:message]    = "Initial Commit"
+      options[:parents]    = []
+      options[:update_ref] = 'HEAD'
```

Comment:
I'm surprised there isn't a master branch.  Does awx/tower only reference the HEAD commit anyway?

Reply:
There is a master branch after everything is set up. Is that just the default behavior?
```
[root@localhost ~]# cd /var/lib/awx_consolidated_source/
[root@localhost awx_consolidated_source]# git status
# On branch master
nothing to commit, working directory clean
[root@localhost awx_consolidated_source]# git log
commit 882ea0dbf707c366de82655bd1b74376ccf1d712
Author: System <system@localhost>
Date:   Mon May 14 19:06:53 2018 +0000

    Initial Commit
[root@localhost awx_consolidated_source]#
```